### ISSUE TITLE
Suppress JavaScript syntax error

### ISSUE THREAD

**Issue by shinnn** [zabute/stylelint-processor-glamorous#1]
### Problem

```js
'use strict';

const {lint} = require('stylelint'); // v9.1.3

lint({
  code: ']',
  codeFilename: 'exmaple.jsx',
  config: {
    processors: 'stylelint-processor-glamorous', // v0.2.0
    rules: {}
  }
}).catch(err => console.error(err.stack));
```

The source `]` is an invalid JavaScript code.

When I run the script above, `lint()` will be rejected with the following error:

```
SyntaxError: Unexpected token (1:0)
    at Parser.pp$5.raise (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp$3.parseExprAtom (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3750:12)
    at Parser.parseExprAtom (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:7238:22)
    at Parser.pp$3.parseExprSubscripts (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:3344:19)
    at Parser.parseMaybeAssign (/Users/shinnn/github/glamorous-bug/node_modules/babylon/lib/index.js:6474:20)
```

This is unexpected for users as this processor changes [the original behavior of `stylelint.lint()` towards syntax errors](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/node-api.md#syntax-errors):

> `stylelint.lint()` does not reject the Promise when your CSS contains syntax errors. It resolves with an object (see [The returned promise](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/node-api.md#the-returned-promise)) that contains information about the syntax error.

### Expected result

I expect stylelint-processor-glamorous to catch all JavaScript syntax errors.

**Comment by zabute** [comment]
Yeah. I didn't notice that. Thanks for reporting.

**Comment by zabute** [comment]
fixed in 0.3.0
